packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -157,53 +156,43 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
-  let {checkboxProps} = useGridSelectionCheckbox({key: null}, state);
   let dragState: DraggableCollectionState;
   if (isListDraggable) {
     dragState = dragHooks.useDraggableCollectionState({
       collection: state.collection,
       selectionManager: state.selectionManager,
-      renderPreview(selectedKeys, draggedKey) {
+      renderPreview(draggingKeys, draggedKey) {
         let item = state.collection.getItem(draggedKey);
-        let itemWidth = domRef.current.offsetWidth;
-        let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
-        let isSelected = state.selectionManager.isSelected(item.key);
+        let isDraggingMultiple = draggingKeys.size > 1;
+
         return (
           <Provider
             {...provider}
-            UNSAFE_className={classNames(listStyles, 'react-spectrum-ListViewItem', 'is-dragging')}
-            UNSAFE_style={{width: itemWidth, paddingInlineStart: 0}}>
-            <div className={listStyles['react-spectrum-ListViewItem-grid']}>
-              <div className={listStyles['react-spectrum-ListViewItem-draghandle-container']}>
-                <div className={listStyles['react-spectrum-ListViewItem-draghandle-button']}>
-                  <ListGripper />
-                </div>
-              </div>
-              {showCheckbox &&
-                <Checkbox
-                  isSelected={isSelected}
-                  UNSAFE_className={listStyles['react-spectrum-ListViewItem-checkbox']}
-                  isEmphasized
-                  aria-label={checkboxProps['aria-label']} />
-              }
-              <SlotProvider
-                slots={{
-                  content: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
-                  text: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
-                  description: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-description']},
-                  icon: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-icon'], size: 'M'},
-                  image: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-image']},
-                  link: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content'], isQuiet: true},
-                  actionButton: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'], isQuiet: true},
-                  actionGroup: {
-                    UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'],
-                    isQuiet: true,
-                    density: 'compact'
-                  },
-                  actionMenu: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actionmenu'], isQuiet: true}
-                }}>
-                {typeof item.rendered === 'string' ? <Content>{item.rendered}</Content> : item.rendered}
-              </SlotProvider>
+            UNSAFE_style={{width: 194, direction}}>
```

Comment:
Possibly should depend on scale? "size-2400" maps to 192px in medium scale, so probably the closest. 

Also noticed that the height of the preview changes depending on the content. List items without a description slot are shorter. Should we make it match the row height?

Reply:
Updated the width to use size-2400.

It does look like the height is slightly smaller than it should be for items without a description. This can be a follow-up after testing